ts/features/itwallet/machine/eid/context.ts

```diff
@@ -1,15 +1,20 @@
-import { StoredCredential } from "../../common/utils/itwTypesUtils";
+import { type StoredCredential } from "../../common/utils/itwTypesUtils";
+import { IssuanceFailure } from "./failure";
+
+export type Identification =
```

Comment:
```suggestion
export type IdentificationMode =
```
This name sound more appropriate to me. What do you think?

Reply:
Yes, `IdentificationContext` sounds good to me. `IdentificationData` is more like something we could get as response, and not as input.